ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -36,33 +36,43 @@ export default function OnboardingMetametrics() {
   const firstTimeSelectionMetaMetricsName =
     firstTimeFlowTypeNameMap[firstTimeFlowType];
 
-  const metricsEvent = useContext(MetaMetricsContext);
+  const trackEvent = useContext(MetaMetricsContext);
```

Comment:
[REQ] in this first PR you'll need to update import statements for the `MetaMetricsContext` object to point to the the `.new` context file. Then in subsequent PRs it'll change back. This is fine, a little more work for you but ultimately easier to review and more safe with checks. Sorry for the churn!

Reply:
Fixed